### PR TITLE
Initial draft of Magento on CentOS 8 guide

### PR DESCRIPTION
**Pull request by JeffreyNovotny**
This is a user guide for hot to install Magento 2.4 on CentOS 8. It should contain the root index.md document as well as 7 images in .png format.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit f1d2c11a4274751e12ef92e8a560690858765b6d

https://deploy-preview-3815--nostalgic-ptolemy-b01ab8.netlify.app

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]
Step 3 : sudo dnf update && sudo dnf upgrade

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]
The recommendation of a 4GB Linode can go in the before you begin you begin section. You dont want the users to get started and securing a 2GB Linode, later to realize that the requirement was a $GB

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]
The command to check the version of Apache that worked on a centos8 Linode was ``` httpd -v``` was not able to check the version with the command listed
Screenshot 2021-02-24 at 12 16 18 PM

The list of modules required is listed in the output format.  This should be a bulleted list or a comma separated list.
Screenshot 2021-02-24 at 12 22 39 PM

The output should be:
Screenshot 2021-02-24 at 12 25 39 PM

Step 3. Install missing modules. Please confirm what should be the command because it results in the following error message:
Screenshot 2021-02-24 at 12 29 05 PM


The command to check if ssl_module is installed ```sudo httpd -t -D DUMP_MODULES | grep ssl```
The command to install ssl_module is ``` sudo dnf install mod_ssl```

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]
In this guide we also need to provide  the links to the guides that have instructions to install apache on Centos 8, MySQL on Centos, php on Centos 8 in the beginning of the respective sections. You may also want to shorten the length of the guide by not repeating the steps that are already there in the respective guides.

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]
After creating the key, edit the /etc/httpd/conf.d/ssl.conf file. Set the values of SSLCertificateFile and SSLCertificateKeyFile to reference the key you created: However, the file name reads: File: /etc/httpd/conf.d/vhost.conf

**Review by Rajakavitha1 — COMMENTED** [review]
Enable the PHP Opcache for better performance. This setting can be found and modified in opcache.ini.

opcach.ini was not found and had to install it with ```sudo dnf install opcache.ini```
Screenshot 2021-02-24 at 4 40 55 PM

File: docs/guides/websites/ecommerce/install-magento-2-4-on-centos-8/index.md [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+
+        mysql -u magento -p
+7.  Record the name of the database and password you created and store this information in a safe place for later.
+8.  If mysql is not already configured to become active at start-up time, add this entry to `systemctl`:
+
+        systemctl enable mysqld
+
+### PHP
+
+Magento recommends the use of PHP 7.4.0 or higher.
```
Comment (Rajakavitha1) [inline docs/guides/websites/ecommerce/install-magento-2-4-on-centos-8/index.md:247]: This is a requirement not a recommendation:

Magento supports PHP 7.3.0 or later. Please read https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]
Please make installing LEMP on CentOS8 Linode guide as a prerequisite in the before you begin section or provide the links to individual Apache, Msql, PHP 7.4  guides in the prerequisite section of Magento.

**Review by andystevensname — APPROVED** [review]

**Comment by andystevensname** [comment]
Closed in favor of https://github.com/linode/docs/pull/4081

**Pull request closed without merging**